app/presenters/publishing_api/payload_builder/access_limitation.rb

```diff
@@ -4,7 +4,7 @@ class AccessLimitation
       def self.for(item)
         return {} unless item.access_limited? && !item.publicly_visible?
 
-        { access_limited: { organisations: item.organisations.pluck(:id).compact } }
+        { access_limited: { organisations: item.organisations.pluck(:content_id).compact } }
```

Comment:
I think it'd be worth squashing this commit with the previous one - otherwise we have this diff between `pluck(:id)` and `pluck(:content_id)`.